<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Converter</class>
 <widget class="QMainWindow" name="Converter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1301</width>
      <height>701</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Converter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>80</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>80</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Currency</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="mainStack">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>1281</width>
       <height>571</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 127);
</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>20</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="lengthInput">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>351</width>
         <height>87</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="lengthOutput">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>200</y>
         <width>351</width>
         <height>87</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="lIn">
       <property name="geometry">
        <rect>
         <x>122</x>
         <y>320</y>
         <width>351</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="lOut">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>310</y>
         <width>351</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>60</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Weight</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="weightOutput">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>200</y>
         <width>351</width>
         <height>87</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="weightInput">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>200</y>
         <width>351</width>
         <height>87</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="wOut">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>310</y>
         <width>351</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="wIn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>310</y>
         <width>351</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>30</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="timeOutput">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>200</y>
         <width>351</width>
         <height>87</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="timeInput">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>351</width>
         <height>87</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="tOut">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>310</y>
         <width>351</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="tIn">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>320</y>
         <width>351</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>40</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Currency</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="currencyInput">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>351</width>
         <height>87</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="currencyOutput">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>200</y>
         <width>351</width>
         <height>87</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cIn">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>300</y>
         <width>351</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cOut">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>300</y>
         <width>351</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
>>>>>>> remotes/origin/image
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
